MAUI/DataForm/Editors.md

```diff
@@ -0,0 +1,1187 @@
+
+You can filter a particular item in the list items by using the `IsEditable` property, this property enables you to type a list item in the combo box editor.
+
+{% tabs %}
+{% highlight C# %}
+
+    dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+    private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+        if (e.DataFormItem != null && e.DataFormItem.FieldName == "City" && e.DataFormItem is DataFormComboBoxItem comboBoxItem)
+        {
+           comboBoxItem.IsEditable = true;
+        }
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+#### Searching words that starts with the input text
+
+Displays all the matches that contain first character of the typed characters in items source of combobox editor. This strategy is case in-sensitive.
+
+{% tabs %}
+{% highlight C# %}
+
+    dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+    private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+        if (e.DataFormItem != null && e.DataFormItem.FieldName == "City" && e.DataFormItem is DataFormComboBoxItem comboBoxItem)
+        {
+            comboBoxItem.TextSearchMode = DataFormTextSearchMode.StartsWith;
+        }
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+#### Searching words that contains the input text
+
+Displays all the matches that contain the typed characters in items source of combobox editor. This strategy is case in-sensitive.
+
+{% tabs %}
+{% highlight C# %}
+
+    dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+    private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+        if (e.DataFormItem != null && e.DataFormItem.FieldName == "City" && e.DataFormItem is DataFormComboBoxItem comboBoxItem)
+        {
+            comboBoxItem.TextSearchMode = DataFormTextSearchMode.Contains;
+        }
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+#### Changing the height of combobox item 
+
+By default the combo box drop down list item height is `400d` , You can change the size for the drop down list item size by using `MaxDropDownHeight` property in the `DataFormItem`.
+
+{% tabs %}
+{% highlight C# %}
+
+    dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+    private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+        if (e.DataFormItem != null && e.DataFormItem.FieldName == "Country" && e.DataFormItem is DataFormComboBoxItem comboBoxItem)
+        {
+            comboBoxItem.MaxDropDownHeight = 400;
+        }
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+#### Loading the complex type property values in combo box editor
+
+You can display the complex type property values in combo box editor by using the DisplayMemberPath and SelectedValuePath properties of DataFormComboBoxItem. You need to use `GenerateDataFormItem `event to set `DisplayMemberPath` and `SelectedValuePath` property value of DataFormComboBoxItem for complex type property.
```

Comment:
You can display the complex type property values in combo box editor by using the `DisplayMemberPath` and `SelectedValuePath` properties of `DataFormComboBoxItem`. You need to use `GenerateDataFormItem `event to set `DisplayMemberPath` and `SelectedValuePath` property values of `DataFormComboBoxItem`.

Reply:
updated